clues.go

```diff
@@ -224,3 +224,63 @@ func AddLabelCounterTo(
 
 	return setNodeInCtx(ctx, namespace, nn)
 }
+
+// ---------------------------------------------------------------------------
+// comments
```

Comment:
nit: could you please fix this copy/paste issue?
```suggestion
// proxies
```

Reply:
d'oh, thanks.